### PR TITLE
DOCS-3867: TrueCommand 2.2 & SCALE Clustering Documentation

### PR DESCRIPTION
**Pull request by Mrt134**
This removes the obsolete 2.1 documentation for the cluster feature and redirects the offline articles to the new content.
- Solutions/Integrations has a new tutorial article for setting up, creating, and sharing cluster volumes.
- TrueCommand has a new UI reference article that shows images and describes each option on the Clusters screen.
- TrueNAS SCALE Getting Started Guide has a new article designed to help readers prepare a newly installed SCALE system for use in a cluster.

Content follows a standardized and single-sourced approach for easier maintenance of obsolete content.

This PR should not be merged until TrueCommand 2.2 is released and 2.1 documentation is ready to be archived.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/Solutions/Integrations/SMBClustering.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+## Creating the Cluster
+
+When the SCALE, AD, and TrueCommand environments are ready, log in to TrueCommand to cluster the SCALE systems.
+
+1. Click the <span class="iconify" data-icon="mdi:server-network"></span> **Clusters** icon in the upper left. Click **NEW CLUSTER** to see the cluster creation options.
+
+   ![CreateClusterSystemsNetwork](/images/TrueCommand/2.2/CreateClusterSystemsNetwork.png "Network Options for Clustered Systems")
+
+2. Enter a unique name for the cluster and open the dropdown to select the systems to include in the cluster.
+3. When each SCALE system is listed, open the **Network Address** dropdown for each system and choose the static IP address from the previously configured subnet dedicated for cluster traffic.
```
Comment (micjohnson777) [inline content/Solutions/Integrations/SMBClustering.md:42]: where are these system listed? Maybe add where they see the listed systems.

File: content/Solutions/Integrations/SMBClustering.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+## Creating Cluster Volumes
+
+1. In the TrueCommand **Clusters** screen, find the cluster to use and click **CREATE VOLUME**.
+   
+   ![ClustersCreateVolumeDetails](/images/TrueCommand/2.2/ClustersCreateVolumeDetails.png "Add Cluster Volume: Details")
+   
+2. Enter a unique name for the cluster and select a **Type**.
+
+   ### Volume Types
```
Comment (micjohnson777) [inline content/Solutions/Integrations/SMBClustering.md:76]: Does this render in the toc? if not, just use the bold emphasis.

File: content/Solutions/Integrations/SMBClustering.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+   {{< expand "Current Types (click to expand)" "v" >}}
+   {{< include file="/content/_includes/ClusterTypes.md" type="page" >}}
+   {{< /expand >}}
+
+3. After configuring the **Type**, enter a **Brick Size** based on the available storage from the clustered pools and your storage requirements.
+4. Review the **Pools** for each SCALE system in the cluster and ensure that the desired pool is used for this cluster volume.
+5. Click **NEXT**.
+6. Review the settings for the new volume and click **CREATE** when ready.
+
+New cluster volumes are added to the individual Cluster cards on the TrueCommand **Clusters** screen.
```
Comment (micjohnson777) [inline content/Solutions/Integrations/SMBClustering.md:87]: why is cluster capitalized? if it is a general mention of a cluster don't capitalize it. If you are talking about something they see on the screen called Cluster use bold.

File: content/TrueCommand/Clusters.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+
+The **Clusters** screen contains all options related to the cluster feature.
+To see this screen, log in to TrueCommand and click the <span class="iconify" data-icon="mdi:server-network"></span> **Clusters** icon in the upper left.
+
+![ClustersScreen](/images/TrueCommand/2.2/ClustersScreen.png "TrueCommand Clusters Screen")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+When no clusters are created, there is a single option on this screen:
+
+| Setting | Value | Description |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:32]: remove the Value column to be consistent with the CORE and SCALE documentation. Or we can do that later when we reorg TC content.

File: content/TrueCommand/Clusters.md [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+|---------|-------|-------------|
+| CREATE CLUSTER | button | Opens the form to create a new cluster. |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:34]: suggest removing the third table column.
```suggestion
| **CREATE CLUSTER** | button | Opens the form to create a new cluster. |
```

File: content/TrueCommand/Clusters.md [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+
+The initial form has two fields:
+
+![CreateClusterSystems](/images/TrueCommand/2.2/CreateClusterSystems.png "Configuring Systems in the Cluster")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+
+| Setting | Value | Description |
+|---------|-------|-------------|
+| Name | string | Identifies the cluster. |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:52]:
```suggestion
| **Name** |  Enter a string to identify the cluster. |
```

File: content/TrueCommand/Clusters.md [hunk 7]
```diff
@@ -0,0 +1,260 @@
...
+
+#### Systems
+
+The initial form has two fields:
+
+![CreateClusterSystems](/images/TrueCommand/2.2/CreateClusterSystems.png "Configuring Systems in the Cluster")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+
+| Setting | Value | Description |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:50]: suggest removing the third table column for consistency with CORE and SCALE docs
```suggestion
| Setting | Description |
```

File: content/TrueCommand/Clusters.md [hunk 8]
```diff
@@ -0,0 +1,260 @@
...
+|---------|-------|-------------|
+| Name | string | Identifies the cluster. |
+| Systems | dropdown | Shows all connected TrueNAS SCALE systems available for this cluster. Clusters can consist of between 3 and 20 individual SCALE systems. |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:53]:
```suggestion
| **Systems** | Dropdown shows all connected TrueNAS SCALE systems available for this cluster. Clusters can consist of between 3 and 20 individual SCALE systems. |
```

File: content/TrueCommand/Clusters.md [hunk 9]
```diff
@@ -0,0 +1,260 @@
...
+Making selections in the **Systems** field adds more options.
+
+![CreateClusterSystemsNetwork](/images/TrueCommand/2.2/CreateClusterSystemsNetwork.png "Network Options for Clustered Systems")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+
+| Setting | Value | Description |
+|---------|-------|-------------|
+| Delete | button | Clicking the  <i class="material-icons" aria-hidden="true" title="Delete">delete</i> **Delete** icon removes the system from the cluster. |
+| Network Address | dropdown | Shows available IP addresses to use for cluster traffic. Using private dedicated network addresses is recommended. |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:67]:
```suggestion
| **Network Address** | Dropdown list shows available IP addresses to use for cluster traffic. Using private dedicated network addresses is recommended. |
```

File: content/TrueCommand/Clusters.md [hunk 10]
```diff
@@ -0,0 +1,260 @@
...
+
+Making selections in the **Systems** field adds more options.
+
+![CreateClusterSystemsNetwork](/images/TrueCommand/2.2/CreateClusterSystemsNetwork.png "Network Options for Clustered Systems")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+
+| Setting | Value | Description |
+|---------|-------|-------------|
+| Delete | button | Clicking the  <i class="material-icons" aria-hidden="true" title="Delete">delete</i> **Delete** icon removes the system from the cluster. |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:66]:
```suggestion
| **Delete** | Clicking the  <i class="material-icons" aria-hidden="true" title="Delete">delete</i> **Delete** icon removes the system from the cluster. |
```

File: content/TrueCommand/Clusters.md [hunk 11]
```diff
@@ -0,0 +1,260 @@
...
+{{< /expand >}}
+
+Making selections in the **Systems** field adds more options.
+
+![CreateClusterSystemsNetwork](/images/TrueCommand/2.2/CreateClusterSystemsNetwork.png "Network Options for Clustered Systems")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+
+| Setting | Value | Description |
+|---------|-------|-------------|
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:65]:
```suggestion
|---------|-------------|
```

File: content/TrueCommand/Clusters.md [hunk 12]
```diff
@@ -0,0 +1,260 @@
...
+
+#### Confirmation
+
+There is a single option on this page.
+
+![CreateClusterReview](/images/TrueCommand/2.2/CreateClusterReview.png "Review and create page for Clusters")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+
+| Setting | Value | Description |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:79]:
```suggestion
| Setting | Description |
```

File: content/TrueCommand/Clusters.md [hunk 13]
```diff
@@ -0,0 +1,260 @@
...
+|---------|-------|-------------|
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:80]:
```suggestion
|---------|-------------|
```

File: content/TrueCommand/Clusters.md [hunk 14]
```diff
@@ -0,0 +1,260 @@
...
+| CREATE | button | Begins the creation process for a cluster. This restricts the SMB functionality on the SCALE systems. |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:81]:
```suggestion
| **CREATE** | Begins the creation process for a cluster. This restricts the SMB functionality on the SCALE systems. |
```

File: content/TrueCommand/Clusters.md [hunk 15]
```diff
@@ -0,0 +1,260 @@
...
+
+#### SMB Network
+
+The SMB Network page has general options and fields added for each SCALE system in the cluster.
+
+![ConfigureClusterSMBNetwork](/images/TrueCommand/2.2/ConfigureClusterSMBNetwork.png "Configure Cluster SMB Network")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+
+| Setting | Value | Description |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:98]:
```suggestion
| Setting | Description |
```

File: content/TrueCommand/Clusters.md [hunk 16]
```diff
@@ -0,0 +1,260 @@
...
+|---------|-------|-------------|
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:99]:
```suggestion
|---------|-------------|
```

File: content/TrueCommand/Clusters.md [hunk 17]
```diff
@@ -0,0 +1,260 @@
...
+| NEXT | button | Continues to the next page. |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:100]:
```suggestion
| **NEXT** | Continues to the next page. |
```

File: content/TrueCommand/Clusters.md [hunk 18]
```diff
@@ -0,0 +1,260 @@
...
+| Skip this step | checkbox | Finalize the cluster configuration without registering a network to use for SMB sharing. Setting this permanently removes the ability to share data stored in this cluster. |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:101]:
```suggestion
| **Skip this step** | Select to finalize the cluster configuration without registering a network to use for SMB sharing. This permanently removes the ability to share data stored in this cluster. |
```

File: content/TrueCommand/Clusters.md [hunk 19]
```diff
@@ -0,0 +1,260 @@
...
+| Interface | dropdown | SCALE system network interface to use for SMB traffic. For best functionality, choose an interface that can communicate between the individual SCALE systems, TrueCommand environment, Active Directory, and and client systems that need to access data shared from this cluster. |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:102]:
```suggestion
| **Interface** | SCALE system network interface to use for SMB traffic. For best functionality, select an interface from the dropdown list that can communicate between the individual SCALE systems, TrueCommand environment, Active Directory, and and client systems that need to access data shared from this cluster. |
```

File: content/TrueCommand/Clusters.md [hunk 20]
```diff
@@ -0,0 +1,260 @@
...
+| Address | dropdown | IP address used for clients to access the SCALE system. |
```
Comment (micjohnson777) [inline content/TrueCommand/Clusters.md:103]:
```suggestion
| **Address** | Select the IP address from the dropdown list used for clients to access the SCALE system. |
```

File: content/_includes/ClusterWarnings.md [hunk 21]
```diff
@@ -0,0 +1,9 @@
+Clustering is a back-end feature in TrueNAS SCALE and should only be configured using the TrueCommand web interface.
+Attempting to configure or manage clustering from within the TrueNAS SCALE UI or Shell can result in cluster failures and permanent data loss.
+
+Using the clustering feature on a SCALE system adds some restrictions to that system:
+
```
Comment (micjohnson777) [inline content/_includes/ClusterWarnings.md:5]: This should also be a bullet list not a numbered list.

**Review by micjohnson777 — COMMENTED** [review]
Just a few things, all of which you can ignore in the interest of time. It will all get caught when we reorg the TC content.

File: content/Solutions/Integrations/SMBClustering.md (hunk 2 above)
Comment (Mrt134) [inline content/Solutions/Integrations/SMBClustering.md:76]:
```suggestion
  **Volume Types**
```

File: content/Solutions/Integrations/SMBClustering.md (hunk 3 above)
Comment (Mrt134) [inline content/Solutions/Integrations/SMBClustering.md:87]:
```suggestion
New cluster volumes are added to the individual cluster cards on the TrueCommand **Clusters** screen.
```

File: content/Solutions/Integrations/SMBClustering.md [hunk 22]
```diff
@@ -0,0 +1,128 @@
...
+The SMB share is added to the SCALE **Shares** > **SMB** section for each system in the cluster.
+Attempting to manage the share from the SCALE UI is not recommended.
+{{< /hint >}}
+
+### Connecting to the Shared Volume
+
+There are many different ways to access an SMB share, but this article demonstrates using the Windows 10 File Explorer.
+
+1. From a Windows 10 system that is connected to the same network as the clustering environment, open File Explorer.
+2. In the Navigation bar, clear the contents and enter `\\` followed by the IP address or hostname of one of the clustered SCALE systems. Press <kbd>Enter</kbd>.
```
Comment (Mrt134) [inline content/Solutions/Integrations/SMBClustering.md:120]:
```suggestion
2. In the **Navigation** bar, clear the contents and enter `\\` followed by the IP address or host name of one of the clustered SCALE systems. Press <kbd>Enter</kbd>.
```

File: content/_includes/ClusterRequirements.md [hunk 23]
```diff
@@ -0,0 +1,10 @@
+To set up clustering with TrueNAS SCALE, you need:
+
+1. 3-20 TrueNAS SCALE systems (version 22.02.2 or later) on the same network. Each SCALE system must have:
+   a. Two network interfaces and subnets.
+      The primary network interface and subnet is used for client access to the SCALE system 
+	  The secondary interface and subnet is dedicated for cluster traffic. This interface must use static IP addresses.
+   b. Disks available or Storage pools already created and available for use.
+2. A TrueCommand 2.2 or later environment that is on the same network as the SCALE systems.
+3. A Microsoft Active Directory environment must be available and connected to the same network as the SCALE systems and TrueCommand environment.
```
Comment (Mrt134) [inline content/_includes/ClusterRequirements.md:9]:
```suggestion
* 3-20 TrueNAS SCALE systems (version 22.02.2 or later) on the same network. Each SCALE system must have:
   * Two network interfaces and subnets.
      The primary network interface and subnet is used for client access to the SCALE system 
	  The secondary interface and subnet is dedicated for cluster traffic. This interface must use static IP addresses.
   * Disks available or Storage pools already created and available for use.
* A TrueCommand 2.2 or later environment that is on the same network as the SCALE systems.
* A Microsoft Active Directory environment must be available and connected to the same network as the SCALE systems and TrueCommand environment.
```

**Review by micjohnson777 — APPROVED** [review]
Two thumbs up!

**Review by micjohnson777 — APPROVED** [review]

**Pull request merged**